qml/EventDetails.qml

```diff
@@ -282,7 +282,7 @@ Page {
         Rectangle{
             id: titleContainer
 
-            color: collection.color
+            color:  { var c = collection.color; c.a = 0.5 ; return c; }
```

Comment:
Please explain the reason of this change. In any case, it might be a good idea to move it to a separate commit, since this seems unrelated.

Reply:
This done as some of the calendars colors are quite bright. i think we can remove it  from this PR.